# Single link in a formation between two drones
string drone_name
string[] targets
geometry_msgs/Vector3[] bearings
std_msgs/Float64[] distances
